mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraCurrentQuotaManager.java

```diff
@@ -47,58 +51,70 @@
     private final PreparedStatement decreaseStatement;
     private final PreparedStatement getCurrentMessageCountStatement;
     private final PreparedStatement getCurrentStorageStatement;
+    private final PreparedStatement getCurrentQuotasStatement;
 
     @Inject
     public CassandraCurrentQuotaManager(Session session) {
         this.session = session;
-        this.increaseStatement = session.prepare(update(CassandraCurrentQuota.TABLE_NAME)
-            .with(incr(CassandraCurrentQuota.MESSAGE_COUNT, bindMarker()))
-            .and(incr(CassandraCurrentQuota.STORAGE, bindMarker()))
+        this.increaseStatement = session.prepare(update(TABLE_NAME)
+            .with(incr(MESSAGE_COUNT, bindMarker()))
+            .and(incr(STORAGE, bindMarker()))
             .where(eq(CassandraCurrentQuota.QUOTA_ROOT, bindMarker())));
-        this.decreaseStatement = session.prepare(update(CassandraCurrentQuota.TABLE_NAME)
-            .with(decr(CassandraCurrentQuota.MESSAGE_COUNT, bindMarker()))
-            .and(decr(CassandraCurrentQuota.STORAGE, bindMarker()))
+        this.decreaseStatement = session.prepare(update(TABLE_NAME)
+            .with(decr(MESSAGE_COUNT, bindMarker()))
+            .and(decr(STORAGE, bindMarker()))
             .where(eq(CassandraCurrentQuota.QUOTA_ROOT, bindMarker())));
-        this.getCurrentMessageCountStatement = session.prepare(select(CassandraCurrentQuota.MESSAGE_COUNT)
-            .from(CassandraCurrentQuota.TABLE_NAME)
+        this.getCurrentMessageCountStatement = session.prepare(select(MESSAGE_COUNT)
+            .from(TABLE_NAME)
             .where(eq(CassandraCurrentQuota.QUOTA_ROOT, bindMarker())));
-        this.getCurrentStorageStatement = session.prepare(select(CassandraCurrentQuota.STORAGE)
-            .from(CassandraCurrentQuota.TABLE_NAME)
+        this.getCurrentStorageStatement = session.prepare(select(STORAGE)
+            .from(TABLE_NAME)
+            .where(eq(CassandraCurrentQuota.QUOTA_ROOT, bindMarker())));
```

Comment:
Small thing but `CassandraCurrentQuota.QUOTA_ROOT` can import static

Reply:
I think I had it before, seems got lost in the multiples rebases... Thanks for spotting it !